PhpAmqpLib/Connection/AMQPSocketConnection.php

```diff
@@ -29,9 +31,11 @@ public function __construct(
         $login_response = null,
         $locale = 'en_US',
         $timeout = 3,
```

Comment:
Naming could be better I believe.
`$timeout` -> `$read_timeout`
`$read_write_timeout` -> `$write_timeout`